app/core/components/ProjectOwnerField/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import { Autocomplete, TextField } from "@mui/material";
+
+interface ProfilesSelectProps {
+  name: string;
+  label: string;
+  owner: object;
+  helperText?: string;
+}
+
+export const ProjectOwnerField = ({
+  name,
+  label,
+  owner,
+  helperText,
+}: ProfilesSelectProps) => {
+  const profiles: readonly any[] = ["diego", "jorge", "jose"];
```

Comment:
We shouldn't have this hardcoded, we have a `Profiles` table that help us with this info